apisix/cli/apisix.lua

```diff
@@ -22,6 +22,7 @@ local apisix_home = "/usr/local/apisix"
 local pkg_cpath = apisix_home .. "/deps/lib64/lua/5.1/?.so;"
                   .. apisix_home .. "/deps/lib/lua/5.1/?.so;"
 local pkg_path = apisix_home .. "/deps/share/lua/5.1/?.lua;"
+                  .. apisix_home .. "/?.lua;"
```

Comment:
`apisix_home` is "/usr/local/apisix", add this would search `/usr/local/apisix/?.lua`, woud this solve probelm?

Reply:
> `apisix_home` is "/usr/local/apisix", add this would search `/usr/local/apisix/?.lua`, woud this solve probelm?

yes, i've verified this in my local docker environment.